src/nvidia-cuda/devcontainer-feature.json

```diff
@@ -44,43 +44,6 @@
       ],
       "default": "11.8",
       "description": "Version of CUDA to install"
-    },
-    "cudnnVersion": {
```

Comment:
We should not delete the Feature option, we should still let user pin to a specific version if they'd like.

Instead, we could update `default` to `automatic`

Reply:
done